### PR TITLE
LPS-97212 Fix how to get default locale requested when an form builder is opened

### PR DESCRIPTION
**Pull request by mclomello**
@natocesarrego could you review this pr please?

### CODE REVIEW

**Comment by mclomello** [comment]
ci:test:sf

**Comment by mclomello** [comment]
ci:test:relevant

**Comment by mclomello** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java [hunk 1]
```diff
@@ -1110,6 +1111,18 @@ protected DDMFormInstanceRecord getDDMFormInstanceRecord() {
 			DDMFormWebKeys.DYNAMIC_DATA_MAPPING_FORM_INSTANCE_RECORD);
 	}
 
+	protected Locale getDefaultLocale() {
+		ThemeDisplay themeDisplay = formAdminRequestHelper.getThemeDisplay();
+
+		User user = themeDisplay.getUser();
+
+		return Optional.ofNullable(
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java]: Good job! 🙂

**Comment by natocesarrego** [comment]
Hey @mclomello,

I just tested you fix locally and I realized that there is a way to set the portal language, through the URL, that isn't being handled by this fix. Please check the image below:

Screen Shot 2019-09-18 at 10 31 01 AM

Can you please make your fix address this use case too?

Thanks in advance.

**Pull request closed without merging**